packages/calypso-color-schemes/bin/postinstall.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-nodejs-modules */
```

Comment:
What do you think about adding `packages/*/bin/**/*` at the overrides ruleset originally meant for regular Calypso bin scripts?

https://github.com/Automattic/wp-calypso/blob/241292989efa898345417b4632f6402605497895/.eslintrc.js#L14-L23

Not a blocker for this PR at any rate.

Reply:
or at `packages/calypso-color-schemes/.eslintrc.js`

Reply:
We could consider an override like that at project level.

The i18n package does have an override, but there are many files involved:

https://github.com/Automattic/wp-calypso/blob/master/packages/i18n-calypso/cli/.eslintrc.js

I don't mind having a few eslint disables inline in the bin files unless you strongly object to that.

Reply:
No strong objections 👍